tests/ha/iscsi_client.pm

```diff
@@ -61,12 +61,16 @@ sub run {
     for (1 .. 40) { send_key 'backspace'; }
     type_string 'iqn.1996-04.de.suse:01:' . get_hostname . '.' . get_cluster_name;
     wait_still_screen 3;
-    send_key 'alt-v';    # discoVered targets
+
+    # According to https://bugzilla.suse.com/show_bug.cgi?id=1231385#c9, change the hotkey for 15-SP6
+    is_sle('=15-SP6') ? send_key 'alt-d' : send_key 'alt-v';    # discovered targets
```

Comment:
Let's make it `is_sle('>=15-SP6')` ... seems the update got into 15-SP7 too: https://openqa.suse.de/tests/overview?distri=sle&version=15-SP7&build=41.1&groupid=143

I also wonder if it would make sense to check the version of `yast2-iscsi-client` instead of the OS version.